src/main/java/springsamurais/toyswapbackend/controller/ListingController.java

```diff
@@ -0,0 +1,28 @@
+package springsamurais.toyswapbackend.controller;
+
+import org.springframework.web.bind.annotation.*;
+import springsamurais.toyswapbackend.model.Listing;
+
+
+@RestController
+@RequestMapping
```

Comment:
Just checking that adding routing is in the next step? i.e. `RequestMapping("api/v1/")` is planned in the next phase? If so, that's fine.